spec/datetimeformat.html

```diff
@@ -1250,6 +1256,9 @@ <h1>Properties of Intl.DateTimeFormat Instances</h1>
       <li>[[DateStyle]], [[TimeStyle]] are each either *undefined*, or a String value with values *"full"*, *"long"*, *"medium"*, or *"short"*.</li>
       <li>[[Pattern]] is a String value as described in <emu-xref href="#sec-intl.datetimeformat-internal-slots"></emu-xref>.</li>
       <li>[[RangePatterns]] is a Record as described in <emu-xref href="#sec-intl.datetimeformat-internal-slots"></emu-xref>.</li>
+      <li>[[NumberFormat1]] is a NumberFormat object used for formatting.</li>
+      <li>[[NumberFormat2]] is a NumberFormat object used for formatting.</li>
+      <li>[[NumberFormat3]] is a NumberFormat object used for formatting.</li>
```

Comment:
Should there be more information here about these three? Also since `[[NumberFormat3]]` may or may not be set based on `fractionalSecondDigits`, I suppose we should mention that too?